v1/transform-global.yaml

```diff
@@ -0,0 +1,227 @@
+info:
+  version: '1.0.0'
+  title: Global Transform API
+  description: Global Transform API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /html/{from}/to/{to}{/provider}:
```

Comment:
This might be easy to confuse with the similar-sounding `POST /transform/html/to/wikitext{/title}{/revision}` entrypoint. Maybe it would make it a bit clearer to use more specific parameter names like `{from_language}` and `{to_language}`, or name the entire entrypoint something like `/html-lang/{from}/to/{to}{/provider}`.